joomla/joomla-cms
# Hide Intro Text not working - blog_item.php

In current stable J! version (4.03) the option to hide **Intro Text** is not working on blog views. **Intro Text** is displayed always regardless of settings.

Pull Request for Issue: -

### Summary of Changes
Added `if-endif` around `echo $this->item->introtext;` in blog_item.php


### Testing Instructions
1. Apply PR.
2. Use Cassoipeia template to minimize effects of 3rd party templates.
3. Create an article with intro text.
4. Set option **Intro Text** in **Global Configuration -> Articles -> Articles panel** to **Hide**.
5. Check if this setting is not overwritten in Menu / Article.
6. Check article in category blog view.


### Actual result BEFORE applying this Pull Request
Intro text will be there regardless of the option selected for **Intro Text**.

![Actual Result](https://user-images.githubusercontent.com/1571585/138189268-b8ddd55b-3215-4b58-a7e7-4672cfdc3d03.png)
### Expected result AFTER applying this Pull Request
Intro text should be hidden for the article.

![Expected Result](https://user-images.githubusercontent.com/1571585/138189363-2d70daea-3caa-4908-8cd2-41167a179e9c.png)

### Documentation Changes Required
No.
